frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu_user.dart

```diff
@@ -45,8 +45,29 @@ class MenuUser extends StatelessWidget {
     String iconUrl = context.read<MenuUserBloc>().state.userProfile.iconUrl;
     if (iconUrl.isEmpty) {
       iconUrl = defaultUserAvatar;
+      final String name = context.read<MenuUserBloc>().state.userProfile.name;
+      final Color nameColor = _generateRandomNameColor(name);
```

Comment:
Instead of using random colors, I prefer providing a color picker for users.

Reply:
Sure, you can support customization of colors later. Would you like me to create a new issue for you?

Reply:
Yeah, sounds good

Reply:
@nurmukhametdaniyar Can you please comment on this issue so that I can assign it to you?

https://github.com/AppFlowy-IO/AppFlowy/issues/2535